frontend/src/components/events/index.jsx

```diff
@@ -18,44 +16,46 @@ class Events extends Component {
     this.props.fetchEvents();
   }
 
-  selectForEdit = id => {
-    let edit_event = this.props.events[id];
-    this.setState({
-      event: edit_event
-    });
-  };
-
   render() {
-    return (
-      <div>
-        <h1>Upcoming Events</h1>
-        <div className="eventbright-results">
-          <EventList>
-            <CardDeck>
-              {this.props.events.map(event => {
-                return (
-                  <EventItems
-                    key={event.id}
-                    name={
-                      event.name
-                        ? event.name.text
-                        : "Young Masterbuilders In Motion Event"
-                    }
-                    description={
-                      event.description ? event.description.text : ""
-                    }
-                    start={event.start ? event.start.local : "TBA"}
-                    end={event.end ? event.end.local : "TBA"}
-                    logo={event.logo ? event.logo.url : logo}
-                    url={event.url}
-                  />
-                );
-              })}
-            </CardDeck>
-          </EventList>
-        </div>
-      </div>
-    );
+    if (this.props.events.length) {
+      const currentTime = Date.now();
+      let recentPast = this.props.events.length - 1;
+      while (
+        recentPast >= 0 &&
+        Date.parse(this.props.events[recentPast].end.local) - currentTime > 0
+      ) {
+        recentPast--;
+      }
+
+      if (recentPast == this.props.events.length - 1) {
+        return (
+          <div>
+            <EventSection events={[]} isUpcoming={true} />
+            Check back again soon for what's is coming up next for YMIM. If you
```

Comment:
i think this section could be styled a bit better

Reply:
Styling and responsiveness added.